### ISSUE TITLE
Autoremove command

### ISSUE THREAD

**Issue by gitdrug** [ScoopInstaller/Scoop#4515]
Is there a command like "apt autoremove" 
To delete unneeded dependinces and if not yall sould implement it

**Comment by rashil2000** [comment]
Nice suggestion!

For this Scoop will need to keep track of whether a package was installed manually or automatically (as a dependency).

**Comment by gitdrug** [comment]
> 
> 
> Nice suggestion!
> 
> For this Scoop will need to keep track of whether a package was installed manually or automatically (as a dependency).

yall sould implement that then

**Comment by pratikpc** [comment]
@gitdrug autoremove is a bit too easy to make mistakes in though

For example, Package A depends on Package B

Just because you remove Package A does not mean you want to remove Package B in quite a few cases as well.

**Comment by pratikpc** [comment]
I think the best approach is to maybe show a list of dangling packages if we ever implement this and let the user choose to take action on their own with the Scoop suggestion being to either remove or to install to remove it from the autoremove list.

**Comment by pratikpc** [comment]
@gitdrug also we are trying to minimise the number of depends present in the code base because for obvious reasons users can choose to install dependencies on their own and not via Scoop.

**Comment by gitdrug** [comment]
> @gitdrug autoremove is a bit too easy to make mistakes in though
> 
> For example, Package A depends on Package B
> 
> Just because you remove Package A does not mean you want to remove Package B in quite a few cases as well.

yeah that happend to me one and it broke my xfce4 making it crash after about 20 minits of using in and i had to redo "apt install xfce4" to stop it crashing

**Comment by gitdrug** [comment]
> I think the best approach is to maybe show a list of dangling packages if we ever implement this and let the user choose to take action on their own with the Scoop suggestion being to either remove or to install to remove it from the autoremove list.

do this that would be good like "scoop dangling" for the command
